installer/templates/phx_single/config/prod.exs

```diff
@@ -26,12 +26,16 @@ config :logger, level: :info
 #       url: [host: "example.com", port: 443],
 #       https: [
 #         port: 443,
+#         ip: {0, 0, 0, 0},
```

Comment:
I think this is Cowboy's default, but it seems good to be explicit, if nothing else to show that we have the opportunity to specify this.